turbopack/crates/turbopack-core/src/resolve/options.rs

```diff
@@ -324,9 +366,16 @@ pub struct ResolvedMap {
 }
 
 #[turbo_tasks::value(shared)]
-#[derive(Clone, Debug)]
+#[derive(Clone)]
 pub enum ImportMapResult {
     Result(ResolvedVc<ResolveResult>),
+    External(RcStr, ExternalType, ExternalTraced),
+    AliasExternal(
```

Comment:
`AliasExternal` vs `PrimaryAlternativeExternal`

Maybe we could use one naming only?